src/layer.rs

```diff
@@ -16,34 +16,197 @@ use crate::{Error, Glyph, Plist};
 static CONTENTS_FILE: &str = "contents.plist";
 static LAYER_INFO_FILE: &str = "layerinfo.plist";
 
+pub(crate) static LAYER_CONTENTS_FILE: &str = "layercontents.plist";
+pub(crate) static DEFAULT_LAYER_NAME: &str = "public.default";
+pub(crate) static DEFAULT_GLYPHS_DIRNAME: &str = "glyphs";
+
+pub type LayerName = Arc<str>;
+
+#[derive(Debug, Clone, PartialEq)]
+pub struct LayerSet {
+    // The first layer is always the 'default layer'.
+    layers: Vec<Layer>,
```

Comment:
Careful! Storing the layer paths in `Layer` lets users redefine e.g. the default glyph dir path, which the spec says must always be "glyphs". Only the name can be changed at will.